README.md

```diff
@@ -517,6 +518,25 @@ var params = {
 text_to_speech.synthesize(params).pipe(fs.createWriteStream('output.wav'));
 ```
 
+### Tone Analyzer
+Use the [Tone Analyzer][tone_analyzer] service to analyze the
+emotion, writing and social tones of a text.
+
+```js
+var watson = require('watson-developer-cloud');
+
+var tone_analyzer = watson.tone_analyzer({
+  username: '<username>',
+  password: '<password>',
+  version: 'v2-experimental'
```

Comment:
The example in the README needs to be updated

See: https://github.com/herchu/nodejs-wrapper/tree/dev#tone-analyzer